com.unity.cinemachine/Editor/Upgrader/CinemachineUpgradeManager.cs

```diff
@@ -392,7 +192,8 @@ void UpgradeAllScenes(Dictionary<int, List<ConversionLink>> conversionLinksPerSc
                 // GML todo: do we need to do this recursively for child GameObjects?
                 SynchronizeComponents(prefabInstance, convertedCopy, m_ObjectUpgrader.ObsoleteComponentTypesToDelete);
 #if CINEMACHINE_TIMELINE
-                timelineManager.UpdateTimelineReference(prefabInstance.GetComponent<CmCamera>(), conversionLink);
+                if (timelineManager != null)
+                    timelineManager.UpdateTimelineReference(prefabInstance.GetComponent<CmCamera>(), conversionLink);
```

Comment:
I'm unhappy about the specific knowledge of CmCamera here, it breaks the abstraction.  We'll deal with this in a subsequent refactor.